app/graphql/types/installment.rb

```diff
@@ -0,0 +1,31 @@
+class Types::Installment < Types::BaseObject
+  implements Types::Interface::WithTimestamps
+
+  description 'Individual media that belongs to a franchise'
+
+  field :id, ID, null: false
+
+  field :release_order, Integer,
+    null: true,
+    description: 'Order based by date released'
+
+  field :alternative_order, Integer,
+    null: true,
+    description: 'Order based chronologically'
```

Comment:
These should be the rank fields from the other PR you made 😅

Reply:
The other PR goes into this one. I just wanted you to check this before I merged that one in here.